packages/backend/test/core/reports/optimism.test.ts

```diff
@@ -0,0 +1,89 @@
+import { AssetId, UnixTime } from '@l2beat/common'
+import { expect } from 'earljs'
+
+import {
+  addOpTokenReport,
+  OP_TOKEN_ID,
+  OP_TOKEN_SINCE_TIMESTAMP,
+  OPTIMISM_PROJECT_ID,
+} from '../../../src/core/reports/optimism'
+import { ReportRecord } from '../../../src/peripherals/database/ReportRepository'
+
+describe(addOpTokenReport.name, () => {
```

Comment:
It needs a test with multiple reports so that only the relevant one gets updated